src/ui/Quickview/QuickviewDocument.ts

```diff
@@ -68,6 +73,10 @@ export class QuickviewDocument extends Component {
     super(element, QuickviewDocument.ID, bindings);
 
     this.options = ComponentOptions.initComponentOptions(element, QuickviewDocument, options);
+    if (!this.options.title) {
+      this.logger.warn('No title was specified');
+    }
```

Comment:
Given that without specifying a value for the title option, the `iframe` element can't have a title, should a warning be logged or not?